corehq/apps/geospatial/utils.py

```diff
@@ -33,6 +34,15 @@ def get_geo_user_property(domain):
     return config.user_location_property_name
 
 
+@quickcache(['domain'], timeout=24 * 60 * 60)
+def get_flag_assigned_cases_config(domain):
+    try:
+        config = GeoConfig.objects.get(domain=domain)
+    except GeoConfig.DoesNotExist:
+        config = GeoConfig()
+    return config.flag_assigned_cases
+
+
```

Comment:
These are great refactor suggestions. 

> GeoConfig.objects.get() is a very simple query on a small table. I'm not sure that quickcache is much benefit here.

Agreed. I am okay with getting rid of it.

> but instead of GeoConfig.objects.get(), I wonder why we didn't use GeoConfig.objects.get_or_create()

My understanding here is that the util functions are just supposed to be a fetch operation (with a default value) instead of creating a one in the database. 

I plan to create a separate PR on top of this for the refactor just to keep things a bit clean as it touches code related to other functionalities. Hope that works.

Reply:
Update:
> the other util functions really only add a single line each, and can probably be dropped too

I ended up keeping this as I feel like it useful considering it is being referenced at a lot of places.
The above made the changes quite less so took it up in this PR itself.

See  https://github.com/dimagi/commcare-hq/pull/35188/commits/613eacaf93a7b71d9f1708c84453cc44562b961f and https://github.com/dimagi/commcare-hq/pull/35188/commits/af93bcb3a6cfb91b2640c6a523e9c2b2b4bf993d